smart_card_connector_app/src/application_unittest.cc

```diff
@@ -106,11 +107,11 @@ class ReaderNotificationObserver final {
 
  private:
   void OnMessageToJs(const std::string& event_name,
-                     Value message_data,
+                     optional<Value> message_data,
                      optional<RequestId> /*request_id*/) {
```

Comment:
should we `GOOGLE_SMART_CARD_CHECK(message_data);` here?

Reply:
Good idea - added an ASSERT here and the CHECK in a couple of other places.